src/Fixer/Phpdoc/PhpdocTrimFixer.php

```diff
@@ -51,11 +51,6 @@ final class Foo {}
      */
     public function getPriority()
     {
-        /*
-         * Should be run after all phpdoc fixers that add or remove tags, or
-         * alter descriptions. This is so that they don't leave behind blank
-         * lines this fixer would have otherwise cleaned up.
```

Comment:
lets keep for now, especially since we've no test that shows this

Reply:
-> https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/tests/AutoReview/FixerFactoryTest.php#L262